src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -27,6 +27,12 @@
           </mat-chip-list>
         </mat-form-field>
         <div class="error-message" *ngIf="isValidationError">{{message}}</div>
+        <div class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark fs-14" *ngIf="!isValidationError" (click)="reflectChange()">
+          <span>Update</span>
+        </div>
+        <div class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark fs-14"  disabled *ngIf="isValidationError" (click)="reflectChange()">
```

Comment:
Hey instead of using two buttons, how about change the add the disbabled class a/c to condition of `isValidationError`?

Reply:
@Sanji515 Fixed :+1: